src/startup_checks.cpp

```diff
@@ -83,6 +85,8 @@ bool AS::Joystick::check_vehicle_type(ros::NodeHandle * nodeH)
 
   if (vehicle_type == LEXUS_RX_450H)
     PublishControl::max_rot_rad = MAX_ROT_RAD_VEHICLE2;
+  else if (vehicle_type == JUPITER_SPIRIT)
+    PublishControl::max_rot_rad = JUPITER_SPIRIT;
```

Comment:
This line doesn't make sense. The `MAX_ROT_RAD_*` variables are defined in `globals.h` and this should be one too. The value should be the maximum rotation range of the steering wheel in Radians.